src/coreclr/pal/tests/palsuite/CMakeLists.txt

```diff
@@ -299,23 +280,6 @@ add_executable_clr(paltests
   c_runtime/swprintf/test7/test7.cpp
```

Comment:
`swprintf*` and `vswprintf*` are only used in 32 places outside of PAL (mostly in disasm.cpp). They can also use in-place conversion and removed in favor of non-w variants.